File: semester 1/lab_questions/Lab_6/GCD_LCM.c
#include<stdio.h>
int main()
{
    int a,b,rem,prod;
    printf("Enter two numbers: ");
    scanf("%d %d",&a,&b);
    prod=a*b; 
    while(b!=0)
    {
        rem=a%b;
        a=b;
        b=rem;
    }
    printf("GCD is: %d\n",a); 
    printf("LCM is: %d\n",prod/a);
    return 0;
}

File: semester 4/DBMS/gcd_lcm.java
public class gcd_lcm{
    public static void gcd(){
        int n1 = 10, n2 = 1500, prd=1;
        for(int i = 1; i<10; i++){
            if(n1%i==0 && n2%i==0){
                prd *= i;
            }
        }
        System.out.println(prd);
    }

    public static void lcm(){
        int n1 = 10, n2 = 15, prd=1;
        for(int i = n1; i<=n1*n2; i=i+n1){
            if(i%n2==0){
                prd = i;
            }
        }
        System.out.println(prd);
    }
    public static void main(String[] args) {
    gcd();
    lcm();     
    }
}